### PR TITLE
GH actions follow up

### PR DESCRIPTION
**Pull request by jomey**
Small updates removing the 'bumpversion' config and updated some docs.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -59,51 +59,48 @@ Get Started!
...
    your new functionality into a function with a docstring, and add the
    feature to the list in README.rst.
-3. The pull request should work for Python 2.6, 2.7, 3.3, 3.4 and 3.5, and for PyPy. Check
-   https://travis-ci.org/scotthavens/weather_forecast_retrieval/pull_requests
-   and make sure that the tests pass for all supported Python versions.
+#. Once opened, every pull request will be tested against every supported
+   Python version through GitHub actions. Check the tab if there are any issues
+   and that all workflows pass.
 
 Tips
```
Comment (scotthavens) [inline CONTRIBUTING.rst:105]: Could you add this:

Releasing to Pypi
-------------------

To create a new release on Pypi.org, follow these steps:

1. Create a new release for `weather_forecast_retrieval`
2. Name the tag and release the version number, for example `v0.7.0`
3. Add documentation about the release and why it's different from the previous
4. Publish new release which will trigger a build to release to Pypi

**Review by scotthavens — APPROVED** [review]
Thanks for cleaning up after me!

**Comment by scotthavens** [comment]
Looks good, thanks for that. I'll make sure to update these for the next conversion.

Probably won't make a release for this as no code was changed.

**Comment by jomey** [comment]
Agreed. Release seems an overkill for this :smile:

**Pull request merged**